docs/general/troubleshooting.md

```diff
@@ -0,0 +1,23 @@
+## Common SSH Issues
+
+Here are some of the most common issues users face when using SSH. 
+
+
+### Keys
+
+The following clusters use SSH keys: Atomate, Farm, Franklin, HPC1, HPC2, Impact, Peloton. 
+
+If you connect to one of these and are asked for a password (as distinct from a passphrase for your key), 
+your key is not being recognized. This is usually because of permissions or an unexpected filename. 
+SSH expects your key to be one of a specific set of names. Unless you have specified something other than
+the default, this is probably going to be **$HOME/.ssh/id_rsa**.
```

Comment:
You can make this console-style by doing `$HOME/.ssh/id_rsa` (with backticks instead of asterisks)